3 - PB/Documentazione esterna/Specifica Tecnica/Specifica Tecnica.typ

```diff
@@ -331,4 +399,112 @@ In nessun caso il database verrà modificato dall'applicazione.
 
 #pagebreak()
 
-= Requisiti soddisfatti ( aggiungere tabella requisiti soddisfatti)
\ No newline at end of file
+= Requisiti soddisfatti
+
+Di seguito vengono riportati i requisti funzionali, di qualità e di vincolo soddisfatti dall'applicazione.
+
+Per una visione più completa sui requisiti si rimanda al documento #adr_v.
+
+== Requisiti funzionali soddisfatti
+
+#show figure: set block(breakable: true)
+#show "#ndp_v": [#ndp_v]
+#show "#pdq_v": [#pdq_v]
+#let requirementsSubset = ()
```

Comment:
Questo mi pare non venga utilizzato in seguito, quindi si può anche togliere, a meno che non serva per qualcosa che non hai ancora pushato

Reply:
doveva servire per una versione precedente, ma posso farne a meno hai ragione